src/components/Dropdown/Dropdown.tsx

```diff
@@ -1,5 +1,41 @@
-export {
-  default as Dropdown,
+import React from 'react'
+import {
+  default as DropdownSemantic,
   DropdownProps
 } from 'semantic-ui-react/dist/commonjs/modules/Dropdown/Dropdown'
+import { Icon } from '../..'
+import { Mobile, NotMobile } from '../Media'
 import './Dropdown.css'
+
+export type DropdownPropsAndTitle = DropdownProps & {
+  title: string
+}
```

Comment:
Can we rewrite this so we export `DropdownProps` as the new type for everyone to use?
We'll need to import `DropdownProps` as `DropdownPropsSemanitc` to make it work.
```suggestion
export type DropdownProps = DropdownPropsSemanitc & {
 /* Mobile header title */
  title: string
}
```